bst_plugin/nst_ppl_surface_full_V1.m

```diff
@@ -0,0 +1,991 @@
+end
+
+
+function [files_in, redone_imports] = import_nirs_files(nirs_fns, subject_names, options)
+files_in = cell(size(nirs_fns));
+redone_imports = zeros(size(nirs_fns));
+for ifile=1:length(nirs_fns)
+    %% Import data
+    nirs_fn = nirs_fns{ifile};
+    if isempty(subject_names{ifile})
+        [root, subject_name, ext] = fileparts(nirs_fn);
+    else
+        subject_name = subject_names{ifile};
+    end
+    condition = ['origin' get_ppl_tag()];
+    [file_in, redone] = nst_run_bst_proc([subject_name '/' condition '/Raw'], options.import.redo, ...
+                                           'process_import_data_time', [], [], ...
+                                           'subjectname',  subject_name, ...
+                                           'condition',    condition, ...
+                                           'datafile',     {nirs_fn, 'NIRS-BRS'}, ...
+                                           'timewindow',   [], ...
+                                           'split',        0, ...
+                                           'ignoreshort',  1, ...
+                                           'channelalign', 1, ...
+                                           'usectfcomp',   0, ...
+                                           'usessp',       0, ...
+                                           'freq',         [], ...
+                                           'baseline',     []);
+    redone_imports(ifile) = redone;
+    %% Manage movement event markings TODO
+    if redone
+        evt_formats = bst_get('FileFilters', 'events');
+        evt_format = evt_formats(strcmp('BST', evt_formats(:,3)), :);
+
+        moco_fn = get_moco_markings_fn(subject_name, options.moco.export_dir);
+        if exist(moco_fn, 'file')
+            % Load event from pre-saved file
+            % TODO: test
+            sFile_in = load(file_fullpath(file_in));
+            [sFile_in, events] = import_events(sFile_in, [], moco_fn, evt_format);   
+        else
+            % Create empty event group
+            movement_events = db_template('event');
+            movement_events.label = 'movement_artefacts';
+            sFile_in = bst_process('GetInputStruct', file_in);
+            process_nst_import_csv_events('import_events', [], sFile_in, movement_events);
+        end
+
+        bad_chans_fn = get_bad_chan_markings_fn(subject_name, options.tag_bad_channels.export_dir);
+        if exist(bad_chans_fn, 'file')
+            % TODO: load content of .mat and set channel flag
+            % TODO: save data -> see process_nst_tag_bad_channels
+        end
+    end
+    %% Manage bad channel markins
+    % TODO: update channel flags
+    % 
+    files_in{ifile} = file_in;
+end
+
+end
+
+
+function redone=load_anatomy(options,i_subject)
+
+    subject=options.import.subject{i_subject};
+    [sSubject, iSubject] = bst_get('Subject',subject.name, 1);
+
+    
+    redone=options.import.redo && length(sSubject.Surface) >= 1;
```

Comment:
At this stage, it's rather a "redo" flag than "redone".
Also I think it should be:

`redo = options.import.redo || isempty(sSubject.Anatomy) || isempty(sSubject.Surface)
`